.cirrus.yml

```diff
@@ -0,0 +1,96 @@
+linux_aarch64_test_task:
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-builder-arm64
+    architecture: arm64
+    platform: linux
+    cpu: 4
+    memory: 16G
+
+  pip_cache:
+    folder: ~/.cache/pip
+
+  test_script: |
+    # apt-get install -yy gcc python3.10 python3-dev
+    python3 -m pip install --upgrade pip setuptools
+    python3 -m venv mpl-dev
+    source mpl-dev/bin/activate
+    python3 -m pip install --upgrade 'contourpy>=1.0.1' cycler fonttools kiwisolver importlib_resources \
+      numpy packaging pillow pyparsing python-dateutil setuptools-scm sphinx
+    python3 -m pip install -r requirements/testing/all.txt
+    python3 -m pip install -ve .
+    python3 -mpytest -raR -n auto --log-level=DEBUG --color=yes
+
+
+macos_arm64_test_task:
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-monterey-xcode:13.3.1
+
+  pip_cache:
+    folder: ~/.cache/pip
+
+  test_script: |
+    brew install python-tk@3.10
+    export PATH=/opt/homebrew/opt/python@3.10/libexec/bin:$PATH
+    python --version
+    pushd ~/
+    python -m venv mpl-dev
+    source mpl-dev/bin/activate
+    popd
+
+    python -m pip install --upgrade pip setuptools
+    python -m pip install --upgrade 'contourpy>=1.0.1' cycler fonttools kiwisolver importlib_resources \
+      numpy packaging pillow pyparsing python-dateutil setuptools-scm sphinx tk
+    python -m pip install -r requirements/testing/all.txt
+    python -m pip install -ve .
+    python -mpytest -raR -n auto \
+      --maxfail=50 --timeout=300 --durations=25 \
+      --log-level=DEBUG --color=yes
+
+
+freebsd_test_task:
+  freebsd_instance:
+    image_family: freebsd-13-1
+
+  pip_cache:
+    folder: ~/.cache/pip
+
+  test_script: |
+    # pkg update -f
+    # pkg upgrade -f
+    pkg install -y python39 py39-pip py39-cycler qhull py39-setuptools \
+      py39-pillow git gmake openlibm
+    python3 --version
+    python3 -m venv mpl-dev
+    . mpl-dev/bin/activate
+    pip3 install --upgrade pip
+    # pip3 install -v numpy
+    pip3 install 'contourpy>=1.0.1'
+    pip3 install fonttools
```

Comment:
This was just to test which module did install numpy as a depdendency (despite installing py39-numpy earlier, although not visible now).